src/simulator/src/modules/Output.js

```diff
@@ -59,15 +56,6 @@ export default class Output extends CircuitElement {
         this.inp1 = new Node(this.bitWidth * 10, 0, 0, this)
     }
 
-    /**
-     * @memberof Output
-     * function to generate verilog for output
-     * @return {string}
-     */
-    generateVerilog() {
```

Comment:
This function generates verilog for the output node, removing it will break that feature.

Reply:
Gotcha. This is good then. Thanks